### ISSUE TITLE
App State 0.74 in DEVELOPMENT

### ISSUE THREAD

**Issue by MoamberRaza** [facebook/react-native#44280]
### Description

When Application is in Development Mode why screen time out wont apply on it (Real Device ) . if we want to call api only when user is on background, how to make it possible. ( IOS )

### Steps to reproduce

-- -- 

### React Native Version

0.74

### Affected Platforms

Runtime - iOS

### Output of `npx react-native info`

```text
info Fetching system and libraries information...
System:
  OS: macOS 14.4.1
  CPU: (16) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
  Memory: 1.41 GB / 16.00 GB
  Shell:
    version: "5.9"
    path: /bin/zsh
Binaries:
  Node:
    version: 19.7.0
    path: ~/.nvm/versions/node/v19.7.0/bin/node
  Yarn:
    version: 1.22.19
    path: /usr/local/bin/yarn
  npm:
    version: 9.6.0
    path: ~/.nvm/versions/node/v19.7.0/bin/npm
  Watchman:
    version: 2023.12.04.00
    path: /usr/local/bin/watchman
Managers:
  CocoaPods:
    version: 1.15.2
    path: /Users/developer/.rbenv/shims/pod
SDKs:
  iOS SDK:
    Platforms:
      - DriverKit 23.4
      - iOS 17.4
      - macOS 14.4
      - tvOS 17.4
      - visionOS 1.1
      - watchOS 10.4
  Android SDK:
    API Levels:
      - "28"
      - "30"
      - "31"
      - "33"
      - "34"
    Build Tools:
      - 28.0.3
      - 29.0.2
      - 29.0.3
      - 30.0.2
      - 30.0.3
      - 31.0.0
      - 33.0.0
      - 33.0.1
      - 33.0.2
      - 34.0.0
    System Images:
      - android-28 | Google APIs Intel x86_64 Atom
      - android-29 | Google APIs Intel x86_64 Atom
      - android-33 | Google APIs Intel x86_64 Atom
      - android-33 | Google Play Intel x86_64 Atom
    Android NDK: 22.1.7171670
IDEs:
  Android Studio: 2023.2 AI-232.10300.40.2321.11668458
  Xcode:
    version: 15.3/15E204a
    path: /usr/bin/xcodebuild
Languages:
  Java:
    version: 19.0.2
    path: /usr/bin/javac
  Ruby:
    version: 3.2.2
    path: /Users/developer/.rbenv/shims/ruby
npmPackages:
  "@react-native-community/cli": Not Found
  react:
    installed: 18.2.0
    wanted: 18.2.0
  react-native:
    installed: 0.74.0
    wanted: 0.74.0
  react-native-macos: Not Found
npmGlobalPackages:
  "*react-native*": Not Found
Android:
  hermesEnabled: true
  newArchEnabled: false
iOS:
  hermesEnabled: true
  newArchEnabled: false
```

### Stacktrace or Logs

```text
--
--
--
```

### Reproducer

domain.com

### Screenshots and Videos

--- --

**Comment by cortinico** [comment]
> When Application is in Development Mode why screen time out wont apply on it (Real Device ) . if we want to call api only when user is on background, how to make it possible. ( IOS )

Could you rephrase your question as I don't full understand it.

**Comment by MoamberRaza** [comment]
when we debugging app with physical device, screen of that device remain active and wont go for sleep.

**Comment by cortinico** [comment]
@MoamberRaza can you provide more context?

**Comment by yahacom** [comment]
@MoamberRaza are you sure it's not phone system settings? I.e. on Android in the Developer Settings menu, we can set Don't turn off the screen while charging connected. A similar thing is present on iOS.
